tutorials/06_Advanced_ChangeInputs.md

```diff
@@ -24,30 +23,57 @@ the name of the needed input data is constructed. It is checked whether those in
 
 The prepared input data is a compressed tar archive file "`.tgz`", which can be opened with software such as [7-Zip](https://www.7-zip.org/), or in terminal by `tar` and `untar` commands. 
 
-Local Input Data
-==================
-If you like to get the input data on your local machine you need to copy your ssh key (`/home/[your_cluster_user_name]/.ssh/id_dsa`) to your computer and adjust your local **.Rprofile** by adding the following line:
-``` r
-options(remind_repos=list("scp://cluster.pik-potsdam.de/p/projects/rd3mod/inputdata/output"=list(username="[your_cluster_user_name]",ssh_private_keyfile="[path_to_your_local_copy_of_your_key]")))
+## Download Input Data Automatically
```

Comment:
This only touches on REMIND input data behind `inputRevision`, but not `CESandGDXversion`.  Probably because this tutorial was copied from MAgPIE which does/did not use gdxes.  But REMIND needs both, and both need to correspond (see remindmodel/development_issues#146), which is the reason multiple directories/connections can be used.

Reply:
But that should be fixed with this PR, right? It tells people to do:
```
export REMIND_repos_scp="scp://cluster.pik-potsdam.de/p/projects/rd3mod/inputdata/output;scp://cluster.pik-potsdam.de/p/projects/remind/inputdata/CESparametersAndGDX"
```
which includes also the CESparametersAndGDX.

Reply:
But doesn't tell them why, or which data to download to local directories for use (further below).